Cython/Compiler/Nodes.py

```diff
@@ -1999,6 +1999,11 @@ def generate_function_definitions(self, env, code):
         # Initialize the return variable __pyx_r
         init = ""
         return_type = self.return_type
+        if isinstance(return_type, PyrexTypes.CConstOrVolatileType) and return_type.is_const:
```

Comment:
```suggestion
        if return_type.is_cv_qualified and return_type.is_const:
```

we have attributes like `is_cv_qualified` that are available on all types to use in place of type checks

Reply:
Thanks, updated